### PR TITLE
optimize: add some notes about using scripts

### PR DESCRIPTION
**Pull request by a364176773**
### Ⅰ. Describe what this PR did
supplementary configuration note information

### CODE REVIEW

**Comment by objcoding** [comment]
See if you can remove unnecessary configuration from conf.txt?

File: script/config-center/README.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 ![Since 1.1.0](https://img.shields.io/badge/Since%20-1.1.0-orange.svg?style=flat-square)
 
+## required attributes 
+
+You only need to keep the following configuration from 'config.txt'. For more configuration information, please visit [seata.io](https://seata.io/)
+
+| server                   | client                                 |
+| ------------------------ | -------------------------------------- |
+| store.mode               | service.vgroupMapping.my_test_tx_group |
+| store.db.driverClassName | service.default.grouplist              |
```
Comment (slievrly) [inline script/config-center/README.md:11]: `service.default.grouplist `  only used in `registry.type=file` mode

Comment (a364176773) [reply script/config-center/README.md]: > `service.default.grouplist ` only used in `registry.type=file` mode

PTAL

File: script/config-center/README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 # Script usage demo
 ![Since 1.1.0](https://img.shields.io/badge/Since%20-1.1.0-orange.svg?style=flat-square)
```
Comment (zjinlei) [inline script/config-center/README.md:2]: 1.2.0

Comment (a364176773) [reply script/config-center/README.md]: PTAL

**Comment by slievrly** [comment]
@a364176773 need a brief description of pr title.

**Pull request merged**